bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/Higgs/ggh012j_5f_NLO_FXFX_M125/ggh012j_5f_NLO_FXFX_M125_customizecards.dat

```diff
@@ -0,0 +1,23 @@
+#put card customizations here (change top and higgs mass for example)
+#set param_card mass 6 173.0
+#set param_card yukawa 6 173.0
+set param_card mass 25 125.0
+set param_card frblock 13 0.0 
+set param_card frblock 15 0.0
+set param_card frblock 11 0.0
+set param_card frblock 2 1.0
+set param_card frblock 22 0.0
+set param_card frblock 10 0.0
+set param_card frblock 9 0.0
+set param_card frblock 8 0.0
+set param_card frblock 17 0.0
+set param_card frblock 3 1.0
+set param_card frblock 23 0.0
+set param_card frblock 21 0.0
+set param_card frblock 19 0.0
+set param_card frblock 14 1.0
+set param_card frblock 20 0.0
+set param_card frblock 12 0.0
+set param_card frblock 18 0.0
+set param_card frblock 16 0.0
+set param_card frblock 1 1000.0
```

Comment:
Hi @qyguo , could you please order restriction blocks from 1 to 23? it's hard to read it, I've tried to compare it with the paramcard_heft.dat included in the UFO model you're using. Just for me to understand, you're setting every coupling to 0 but kHgg (+ lambda, cos(a) and kSM) right? I was wondering about kHzz, but since you're decaying the Higgs boson with JHUGen I guess it's ok

Reply:
Thanks. In the new commit, the restriction blocks are ordered.
Yes we set the coupling to 0 except the kHgg lambda cos(a) and kSM.